app/src/main/java/com/github/se/eventradar/model/Location.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.se.eventradar.model
+
+data class Location(
+    var latitude: Double,
+    var longitude: Double,
+    var address: String,
+)
+
+fun getLocation(name: String, lat: Double, lng: Double): Location {
+  return Location(lat, lng, name)
+}
```

Comment:
nit: function name could be clearer, at first glance I didn't understand it was to create a new Location instance from values. I think "LocationFrom()" or "CreateLocation()" or any idea you have.

Reply:
okay done